File: Java/q1011.java
// Esfera

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        double raio = reader.nextDouble();
        double pi = 3.14159;
        double vol = (4/3.0) * pi * (raio*raio*raio);
        
        System.out.printf("VOLUME = %.3f%n", vol);
    }
}

File: Cpp/q1011.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;
 
int main() {
 
    double raio, pi, r, vol;
    pi = 3.14159;
    cin >> raio;
    
    r = pow(raio, 3);
    vol = (4.0/3 * pi * r);
    
    cout << fixed << setprecision(3);
    cout << "VOLUME = " << vol << endl;
 
    return 0;
}